### PR TITLE
Change python package version and improve utility usability

### PR DESCRIPTION
**Pull request by psycoder-sup**
Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/TBD-Labs-AI/latched/blob/repo-setting/CONTRIBUTING.md),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

# Changes
- update `utils/__init__.py` to import utility by `from latcehd.utils import profile`
- Update python version to `3.11.9` 
- Update coremltools version to '8.0b2'

### CODE REVIEW

File: src/latched/utils/models.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 TBD Labs Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import torch.nn as nn
+
+
+def get_model_size(model: nn.Module, unit: str = "gb") -> float:
```
Comment (dev-sungman) [inline src/latched/utils/models.py:7]: Is this valid to check the model size? 

It seems better: https://pytorch.org/tutorials/recipes/recipes/dynamic_quantization.html

```python3
def print_size_of_model(model, label=""):
    [torch.save](https://pytorch.org/docs/stable/generated/torch.save.html#torch.save)(model.state_dict(), "temp.p")
    size=os.path.getsize("temp.p")
    print("model: ",label,' \t','Size (KB):', size/1e3)
    os.remove('temp.p')
    return size

# compare the sizes
f=print_size_of_model(float_lstm,"fp32")
q=print_size_of_model(quantized_lstm,"int8")
print("{0:.2f} times smaller".format(f/q))
```

Comment (psycoder-sup) [reply src/latched/utils/models.py:7]: I'll change it to that one

Comment (psycoder-sup) [reply src/latched/utils/models.py:10]: @sungmanch Add this print function

File: src/latched/utils/models.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    size_all_gb = size_all_mb / 1024
+    if unit == "mb":
+        return size_all_mb
+    elif unit == "gb":
+        return size_all_gb
+    else:
+        raise ValueError(f"Invalid unit: {unit}. Please specify 'mb' or 'gb'.")
+
+
+def print_size_of_model(model: nn.Module, label: str = ""):
```
Comment (dev-sungman) [inline src/latched/utils/models.py:36]: If `get_model_size()` function works well, `print_size_of_model` isn't needed. I just recommend this way since you reported the same model size even though it was quantized.

File: src/latched/utils/models.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+def get_model_size(model: nn.Module, unit: str = "gb") -> float:
+    """Get the size of the model in MB or GB.
+
+    Args:
+        model (nn.Module): The model to get the size of.
+        unit (str): The unit to return the size in. Defaults to "gb".
+
+    Returns:
+        float: The size of the model in MB or GB.
+    """
```
Comment (dev-sungman) [inline src/latched/utils/models.py:19]: Could you add this line? Quantization maybe works well with the eval mode.
```model .eval()```

File: src/latched/utils/models.py (hunk 2 above)
Comment (dev-sungman) [reply src/latched/utils/models.py:36]: Please revmoe this function and check the above comment, instead

**Review by dev-sungman — COMMENTED** [review]
Thanks for the applying, please check the comment

File: src/latched/utils/models.py (hunk 2 above)
Comment (psycoder-sup) [reply src/latched/utils/models.py:36]: I should compare outputs of those two codes

**Comment by psycoder-sup** [comment]
@sungmanch Please check the updated files. I've accepted your suggestion.

**Quantization Takeaways**
- I've tested comparing model size using pytorch's official way of comparing model size. Still both model, original and quantized model, have the same size.

**Review by sungmanch — APPROVED** [review]
LGTM
